src/components/ClientCard/ClientCard.css

```diff
@@ -14,10 +14,81 @@
   border: 0;
   outline: 0;
   cursor: pointer;
-
+  display: flex;
 }
 
-
+.ClientCard.active .ClientCard__btn,
```

Comment:
proper BEM would be `.ClientCard--active` rather than `.ClientCard.active`. "active" in this case is a modifier.

Reply:
too tru too tru
![](http://feelgrafix.com/data_images/out/18/923703-alpaca.jpg)